site/sass/layouts/_tech-detail.scss

```
@@ -36,36 +39,54 @@
   }
 
   &__hero {
-    background: linear-gradient(18deg, var(--grey-100), var(--grey-100) 50%, var(--grey-50) 50%);
     grid-column: 1 / -1;
     grid-row: 2 / span 1;
+    overflow: hidden;
 
     @include mq($bkpt-tech-details--2-cols) {
       grid-column: 2 / -1;
       grid-row: 1 / span 2;
     }
 
-    @include mq($bkpt-tech-details--landing) {
-      min-height: 10rem;
+    &:not(:empty) {
+      @include mq($bkpt-tech-details--2-cols) {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+      }
     }
 
-    &:not(:empty) {
-      display: flex;
-      justify-content: center;
-      padding: 1rem;
+    &-title {
+      color: var(--white);
+      padding: $block-padding clamp(1.5rem, 4vw, 7.5rem);
+
+      @include mq($bkpt-tech-details--2-cols) {
+        align-items: center;
+        display: flex;
+        justify-content: center;
+        padding-bottom: 0;
+        padding-right: 0;
+        padding-top: 0;
+      }
     }
 
     &-image {
-      max-height: 20vh;
-      width: auto;
+      display: flex;
+      height: 25vh;
+      justify-content: flex-end;
+      padding-left: 2rem;
+
+      @include mq($bkpt-tech-details--2-cols) {
+        height: 100%;
+        padding-left: 0;
+      }
 
-      @include mq($bkpt-tech-details--landing) {
-        padding-bottom: 4rem;
+      @include mq($bkpt-tech-details--3-cols) {
+        height: 40vh;
```

I think this needs to be `max-height` or the background image starts to break

```suggestion
        max-height: 40vh;
```